x-pack/test/cases_api_integration/security_and_spaces/tests/common/user_profiles/get_current.ts

```diff
@@ -0,0 +1,64 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import expect from '@kbn/expect';
+import { loginUsers, suggestUserProfiles } from '../../../../common/lib/user_profiles';
+import { FtrProviderContext } from '../../../../common/ftr_provider_context';
+import { superUser } from '../../../../common/lib/authentication/users';
+import { createCase, deleteAllCaseItems } from '../../../../common/lib/utils';
+import { getPostCaseRequest } from '../../../../common/lib/mock';
+import { getUserInfo } from '../../../../common/lib/authentication';
+
+// eslint-disable-next-line import/no-default-export
+export default function ({ getService }: FtrProviderContext) {
+  const supertestWithoutAuth = getService('supertestWithoutAuth');
+  const es = getService('es');
+
+  describe('user_profiles', () => {
+    describe('get_current', () => {
+      afterEach(async () => {
+        await deleteAllCaseItems(es);
+      });
+
+      it('sets the profile uid for a case', async () => {
+        const cookies = await loginUsers({ supertest: supertestWithoutAuth, users: [superUser] });
+
+        const profiles = await suggestUserProfiles({
+          supertest: supertestWithoutAuth,
+          req: {
+            name: 'superUser',
+            owners: ['securitySolutionFixture'],
+            size: 1,
+          },
+          auth: { user: superUser, space: null },
+        });
+
+        const caseInfo = await createCase(
+          supertestWithoutAuth,
+          getPostCaseRequest(),
+          200,
+          { user: superUser, space: null },
+          { Cookie: cookies[0].cookieString() }
```

Comment:
If I don't pass the cookie here then the `getCurrent` call in `userProfile` service of the security plugin returns null because it can't find the session.

@azasypkin is it expected that we need to pass in the cookie for `getCurrent` to return the profile given the kibana request?

Reply:
Ah thanks, yeah I didn't realize we weren't supporting HTTP auth initially 👍 . Thanks for the feedback I'll switch it to only use cookies.